app/tests/spec/models/email-resend.js

```diff
@@ -0,0 +1,103 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+
+define([
+  'chai',
+  'sinon',
+  'models/email-resend',
+  'lib/constants'
+],
+function (chai, sinon, EmailResend, Constants) {
+  var assert = chai.assert;
+
+  describe('models/email-resend', function () {
+    var emailResend;
+
+    afterEach(function () {
+      emailResend = null;
+    });
+
+    describe('incrementRequestCount', function () {
+      it('increments the number of tries', function () {
+        emailResend = new EmailResend();
+        assert.equal(emailResend.get('tries'), 0);
+        emailResend.incrementRequestCount();
+        assert.equal(emailResend.get('tries'), 1);
+      });
+
+      it('calls onTriesExceeded when tries are exceeded', function () {
+        var onTriesExceeded = sinon.spy();
+
+        emailResend = new EmailResend();
+        emailResend.on('maxTriesExceeded', onTriesExceeded);
+
+        for (var i = 0; i < Constants.EMAIL_RESEND_MAX_TRIES; i++) {
+          emailResend.incrementRequestCount();
+        }
+        assert.equal(onTriesExceeded.callCount, 0);
+        emailResend.incrementRequestCount();
+        assert.equal(onTriesExceeded.callCount, 1);
+        for (i = 0; i < 10; i++) {
+          emailResend.incrementRequestCount();
+        }
+        assert.equal(onTriesExceeded.callCount, 11);
+      });
+
+      it('calls onTriesExceeded according to the number of maxTries', function () {
```

Comment:
> We might need to create those resend-mixin.js tests, I cannot find one at this moment :mag:

There is a good chance they are part of the normal views, or we don't have any and the coverage tool gave us false confidence that the logic was tested.